conan/tools/gnu/autotoolstoolchain.py

```diff
@@ -123,8 +123,9 @@ def defines(self):
     def environment(self):
         env = Environment()
         if is_msvc(self._conanfile):
-            env.define("CXX", "cl")
-            env.define("CC", "cl")
+            env.define("CXX", "cl -nologo")
+            env.define("CC", "cl -nologo")
+            env.define("LD", "link -nologo")
```

Comment:
Actually, I'm wondering whether conan should not first check whether CC, CXX and LD are set in profile (for example a user may have several Visual Studio installation, and point to a specific one).
For each of them, if it's set, transform the path to unix_path and add `-nologo`, if not set to `cl -nologo` etc.